openedx/adg/admin_dashboard/performance/views.py

```diff
@@ -155,3 +160,96 @@ def _get_course_key(self):
         """
         return CourseKey.from_string(self.kwargs['course_key_string'])
+
+
+class PerformanceTrackingView(EnsureAdminUserAuthenticatedMixin, ListView):
+    """
+    View for the Performance dashboard.
+    """
+
+    template_name = 'adg/admin_dashboard/performance/performance_tracking.html'
+    has_adg_admin_permissions = False
+    business_line_selected_by_user = ''
+    business_lines = []
+    paginate_by = 10
+
+    def get_business_lines(self):
+        """
+        Select the business line(s) for which data is to be displayed on the performance dashboard
+        """
+        user = self.request.user
+        business_lines = []
+
+        if self.has_adg_admin_permissions and self.business_line_selected_by_user == ALL_BUSINESS_UNITS:
+            business_lines = BusinessLine.objects.all()
+
+        elif self.has_adg_admin_permissions and self.business_line_selected_by_user != ALL_BUSINESS_UNITS:
+            business_lines = BusinessLine.objects.filter(title=self.business_line_selected_by_user)
+
+        elif BusinessLine.is_user_business_line_admin(user):
+            group_list = user.groups.values_list('name', flat=True)
+            business_lines = BusinessLine.objects.filter(title__in=group_list)
+
+        return business_lines
+
+    def get_context_data(self, **kwargs):  # pylint: disable=arguments-differ
+        """
+        Prepare and return the context required for performance dashboard
+
+        Returns:
+            dict: extra context
+        """
+        context = super().get_context_data(**kwargs)
+
+        business_line_titles = list(BusinessLine.objects.get_title_of_business_lines())
+        business_line_titles.append(ALL_BUSINESS_UNITS)
+
+        course_groups = MultilingualCourseGroup.objects.all_prereq_course_groups(self.business_lines)
+
+        context.update({
+            'has_adg_admin_permissions': self.has_adg_admin_permissions,
+            'business_lines': business_line_titles,
+            'selected_business_line': self.business_line_selected_by_user,
+            'accepted_students': UserApplication.objects.get_accepted_students(self.business_lines),
+            'program_completions': ApplicationHub.objects.get_program_completions(self.business_lines),
+            'chart': self.get_data_for_all_course_groups(course_groups),
+        })
+
+        return context
+
+    def get_queryset(self):
+        """
+        Return the program prerequisite courses, common business line prerequisite courses,
+        and user selected business line(s) courses.
+
+        Returns
+            list: Prerequisite courses
+        """
+        self.has_adg_admin_permissions = is_adg_admin(self.request.user)
+        self.business_line_selected_by_user = self.request.GET.get('business-line', '')
+
+        if self.has_adg_admin_permissions and not self.business_line_selected_by_user:
+            self.business_line_selected_by_user = BusinessLine.objects.all().first().title
```

Comment:
In case a user is an adg admin, and has not selected a particular business unit, shouldn't the default be `All Business Units`.